src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/GameResults/GameResultViewModel.cs

```diff
@@ -77,15 +77,39 @@ public GameResultViewModel()
         public int Round { get; set; }
 
         /// <summary>
-        /// Gets or sets the number of the game in the tournament
+        /// Gets or sets the number of the game in the tournament.
         /// </summary>
         public byte GameNumber { get; set; }
 
+        /// <summary>
+        /// Gets or sets numbers of game in which home team has won
+        /// in order to proceed to current game.
+        /// </summary>
+        public byte HomeTeamPreviouslyWonGameNumber { get; set; }
+
+        /// <summary>
+        /// Gets or sets numbers of game in which away team has won
+        /// in order to proceed to current game.
+        /// </summary>
+        public byte AwayTeamPreviouslyWonGameNumber { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whenever we take only team names,
+        /// not game number.
+        /// </summary>
+        public bool MustDisplayOnlyTeamNames { get; set; }
```

Comment:
Why do you need those?

Reply:
MustDisplayOnlyTeamNames :
"Winner#" is going to be displayed of "Team#" only if it's not (1) first round and if it's not (2) bronze match.
It's possible to check for *1 inside partial view _PlayoffGameResult, but impossible to check for *2.
So I needed some indicator in GRVM which tells if we need to extract "Team#" form UI Resources instead of "Winner#".

Home/AwayTeamPreviouslyWonGameNumber :
It was defined in issue that we need to display team as a "Winner{number}", so I tried a few ways to store this "number." As a field it failed, because we instantiate GRVM every time, so it would be just displayed repeatedly the same 2 lines  "Winner1 <br>Winner2". Making it as a static member failed (reloading page was increasing counter).
So, in order to display these 2 numbers I decided to pass them in GRVM.

UPD: But now I think of different outcome : I could just simply add 2 properties (HomeTeamName, AwayTeamName) and, before passing a GRVM instance to _PlayoffGameResult, I could just assing proper names. In this case I would need only 2 additional properties instead of 5, and their assigning would depend only on _PlayoffSchedule.

Reply:
Yep

Reply:
Ok.